pkg/config/config.go

```diff
@@ -14,6 +14,12 @@ type Config struct {
 	dataDir      string // directory managed by Dad (languages distribs, virtualenvs...)
 }
 
+func NewTestConfig() *Config {
+	return &Config{
+		DebugEnabled: false,
+	}
+}
```

Comment:
What about `NewDebugConfig` ?

Reply:
I wanted to have a constructor for very simple config to use in tests (hence the name).
Not sure what you have in mind with this `NewDebugConfig`, but this can change as soon as we have other use cases.